### PR TITLE
Revert "Fixing the read map function for the specific mod2svat.inp fo…

### PR DESCRIPTION
**Pull request by JoerivanEngelen**
This reverts commit 0097d12c7c48ac5b7acd4f8901ba151f3a2f8122.

I discussed this with Robert, and the strip model example has a mod2svat.inp with a format that does not really conform to what is "normal" for MetaSWAP. All files I've seen, except the mod2svat.inp in the strip model, pad with spaces and do not use comma'
s and quotes as seperators. 

I think this specfic mod2svat.inp file in the strip model creeped in during the development of the coupler script, as it is easier to parse with regex, which was used initially to read input files in this python script. 

It is therefore probably better to fix the mod2svat.inp file in the strip model example instead and conform to what is usual than to force others to stick to our specific format.

tl;dr: It is best to revert this commit ;-)

### CODE REVIEW

**Comment by rleander** [comment]
> This reverts commit 0097d12.
> I discussed this with Robert, and the strip model example has a mod2svat.inp with a format that does not really conform to what is "normal" for MetaSWAP. All files I've seen, except the mod2svat.inp in the strip model, pad with spaces and do not use comma'
> s and quotes as seperators.
> I think this specfic mod2svat.inp file in the strip model creeped in during the development of the coupler script, as it is easier to parse with regex, which was used initially to read input files in this python script.
> It is therefore probably better to fix the mod2svat.inp file in the strip model example instead and conform to what is usual than to force others to stick to our specific format.
> tl;dr: It is best to revert this commit ;-)

The assumption that this is a newly produced format (i.e. comes with the setup of the script) seems to be disproven by inspection of Paul's original strip model, of which the input files are stored at

https://repos.deltares.nl/repos/DSCTestbench/trunk/cases/e150_metaswap/f01_basic_tests/c01_ModStrip1/testunsa/inp ... also the code in SIMGRO seems to be prepared  `READ(iun,'(i10,2x,i10,i5)',IOSTAT=ios) mf_r,k_r,ly_r`:

**Comment by JoerivanEngelen** [comment]
Ah interesting! So that hypothesis goes out of the window.

I'm not proficient in FORTRAN, but it seems to me that MetaSWAP does not care about what characters are used as seperators. So whether it is:
```
         2,'         1    1'
         3,'         2    1'
```

or:

```
         2           1    1 
         3           2    1 
```

In my test model, MetaSWAP handled the latter format fine.
However, the iMOD-converter writes files only in the latter format, and for reading and writing files in Python, it is also a lot easier to use the latter format. So I am still inclined towards supporting the latter format in our coupler script. I also cannot find in the IO manual why you would use these comma's and quotes as seperators.

**Comment by rleander** [comment]
The 2X in the read format absorbs the 
,'

or anything. So effectively with the 2X fortran reads the same line as if it were without the punctuation marks. That's why it works. I didn't even consider it yesterday, just modified the script to make it work, because I was in a rush. But I agree with you it is better to leave it this way. Furthermore, the last column lost its function I think... we are not using it.
